numpy/distutils/system_info.py

```diff
@@ -449,6 +449,10 @@ class NotFoundError(DistutilsError):
     """Some third-party program or library is not found."""
 
 
+class AliasedOptionError(DistutilsError):
+    """Aliases entries in config files should not be existing. In section '{section}' we found multiple of options {options} """
+
+
```

Comment:
Needs reformat to 80 column width. Also "mulitple of options" might be better as "multiple appearances of option"